pkgs/by-name/di/disnix-web-service/package.nix

```diff
@@ -1,7 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, ant, jdk, xmlstarlet, axis2, dbus_java }:
-
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  ant,
+  jdk8,
```

Comment:
Please reverse this part. It breaks backwards compatibility with the override signature from the previous, meaning derivations which performed an `.override { jdk = ...; }` will no longer function.

Instead, leave `jdk` hanging and set it in the original `pkgs/top-level/all-packages.nix`. You can see more information and how and why this is done in the `pkgs/by-name/README`

Reply:
yeah makes sense thanks